src/DynamoCore/Graph/Annotations/AnnotationModel.cs

```diff
@@ -331,6 +332,23 @@ public bool IsExpanded
             {
                 isExpanded = value;
                 UpdateBoundaryFromSelection();
+                UpdateErrorAndWarningIconVisibility();
+            }
+        }
+
+        private ElementState groupState = ElementState.Active;
+
+        /// <summary>
+        /// Indicates whether the group contains nodes that are in an info/warning/error state.
+        /// This includes the state of any nodes that are in nested groups.
+        /// </summary>
+        public ElementState GroupState
```

Comment:
@OliverEGreen @reddyashish is this supposed to be serialized to the .dyn? Should it have a [JsonIgnore] attribute applied?

Reply:
It doesn't need to be serialised. 
Will add tomorrow

Reply:
Added!